crates/engine/src/lib.rs

```diff
@@ -135,7 +151,7 @@ impl<T: Default + 'static> Builder<T> {
             let core = c.clone();
             let module = match c.source.clone() {
                 ModuleSource::FileReference(p) => {
-                    let module = Module::from_file(&self.engine, &p).with_context(|| {
+                    let module = Module::from_file(&self.engine.0, &p).with_context(|| {
```

Comment:
Sorry for the noise. I reverted my own feedback to use `.inner` here; doesn't make as much sense with `inner` doing that `clone`.